README.markdown

```diff
@@ -332,23 +318,12 @@ $ make shell
 #### Database
 
 **Tip**: You can connect to the app's database to inspect its contents by
-installing [psql](https://www.postgresql.org/docs/current/static/app-psql.html)
-and then running:
+running:
 
 ```bash
-$ make psql
+$ make sql
 ```
 
-**Tip**: If you want to delete all your data and reset your dev database,
-an easy way to do so is just to delete the whole docker container:
-
-```bash
-$ sudo docker rm lms-postgres
-```
-
-You can then re-create the container by re-running the `docker run` command
-above.
-
```

Comment:
You would just run `docker-compose down -v` but I don't think we need to document `docker-compose` here